qm.te

```diff
@@ -29,3 +29,12 @@ files_pid_filetrans(init_t, ipc_var_run_t, dir, "ipc")
 unconfined_domain(ipc_t)
 
 qm_domain_template(qm)
+
+optional_policy(`
+	require{
```

Comment:
Do you think this doc should be updated also? 
https://github.com/containers/qm/blob/main/qm_selinux.8

Reply:
I think that make sense, yes. I'll update the documentation as soon as we verified these changes work as expected. But [as mentioned here](https://github.com/containers/qm/pull/703#issuecomment-2589794057), I'd need a bit of help.